framework/src/test/java/org/tron/keystore/WalletFileTest.java

```diff
@@ -19,10 +18,59 @@ public void testGetAddress() throws NoSuchAlgorithmException, CipherException {
         SecureRandom.getInstance("NativePRNG"),true));
     WalletFile walletFile2 = Wallet.createStandard("", SignUtils.getGeneratedRandomSign(
         SecureRandom.getInstance("NativePRNG"),true));
-    Assert.assertTrue(!walletFile1.getAddress().equals(walletFile2.getAddress()));
-    Assert.assertTrue(!walletFile1.getCrypto().equals(walletFile2.getCrypto()));
-    Assert.assertTrue(!walletFile1.getId().equals(walletFile2.getId()));
-    Assert.assertTrue(walletFile1.getVersion() == walletFile2.getVersion());
+    WalletFile walletFile3 = (WalletFile) getSame(walletFile1);
```

Comment:
Please add test cases related to the changes.

Reply:
Yes, it shouldn't exist here, I will fix it after https://github.com/tronprotocol/java-tron/pull/5451 is merged.